src/datepicker/datepicker.js

```diff
@@ -161,14 +161,24 @@ angular.module('ui.bootstrap.datepicker', ['ui.bootstrap.dateparser', 'ui.bootst
 
   this.init = function(ngModelCtrl_) {
     ngModelCtrl = ngModelCtrl_;
-    ngModelOptions = ngModelCtrl_.$options ||
-      $scope.datepickerOptions.ngModelOptions ||
-      datepickerConfig.ngModelOptions;
+
+    // grab any timezone value based on precedence
+    var timezone = ngModelCtrl_.$options.getOption('timezone') ||
+      ($scope.datepickerOptions.ngModelOptions ? $scope.datepickerOptions.ngModelOptions.timezone : null) ||
+      (datepickerConfig.ngModelOptions ? datepickerConfig.ngModelOptions.timezone : null);
+
+    // need to start with existing ModelOptions but also give it greatest precedence minus its default of null timezone
+    ngModelOptions = ngModelCtrl_.$options
+      .createChild(datepickerConfig.ngModelOptions)
```

Comment:
This breaks compatibility with 1.5 I believe - we cannot have that for a minor version bump. Is there no way to made this compatible with both versions?